### ISSUE TITLE
problem on view camera

### ISSUE THREAD

**Issue by platini76** [SeydX/homebridge-camera-ui#67]
I have a problem on view camera.
since yesterday I used https://github.com/Sunoo/homebridge-camera-ffmpeg

now, when I switch on your plugin my camera is not active.

my previuos config:
 {
            "name": "Camera FFmpeg",
            "cameras": [
                {
                    "serialNumber": "ZZZZZZZZ",
                    "videoConfig": {
                        "vcodec": "h264",
                        "source": "-rtsp_transport tcp -re -i rtsp://XXXXXXXX@192.168.0.58/h264_stream",
                        "debug": true,
                        "maxFPS": 30,
                        "maxHeight": 720,
                        "maxStreams": 2,
                        "maxWidth": 1280,
                        "audio": true
                    },
                    "model": "Ezviz Mini Plus",
                    "name": "Cam Conigli"
                }
            ],
            "platform": "Camera-ffmpeg"
        },

my actual config:
        {
            "name": "CameraUI",
            "port": 8282,
            "auth": "form",
            "theme": "auto",
            "language": "auto",
            "debug": false,
            "reset": false,
            "mqtt": {
                "active": false
            },
            "http": {
                "active": false,
                "localhttp": false
            },
            "ssl": {
                "active": false
            },
            "cameras": [
                {
                    "name": "Cam Conigli",
                    "manufacturer": "Ezviz",
                    "model": "Ezviz Mini Plus",
                    "serialNumber": "xxxxxx",
                    "unbridge": false,
                    "videoConfig": {
                        "source": "-rtsp_transport tcp -vcodec h264_mmal -i rtsp://admin:UQVWSG@192.168.0.58/Streaming/Channels/1",
                        "stillImageSource": "-i rtsp://xxxxxxxxx@192.168.0.58/Streaming/Channels/1/picture",
                        "maxWidth": 1920,
                        "maxHeight": 1080,
                        "maxFPS": 15,
                        "maxBitrate": 500,
                        "vcodec": "h264_omx",
                        "packetSize": 564,
                        "audio": true
                    }
                }
            ],
            "platform": "CameraUI"
        }

rstp stream worked ... tested.. but no live view... where I am wrong?

**Comment by platini76** [comment]
this is the error with webpage:

[6/12/2020, 23:20:01] [CameraUI] [Interface] Cam Conigli: RTSP stream exited with error! (null)


and with homekit

[6/12/2020, 23:22:18] [CameraUI] [Plugin] Cam Conigli: ffmpeg exited with code: 1 and signal: null (Error)

**Comment by platini76** [comment]
here another log:

[6/12/2020, 23:30:00] [CameraUI] [Interface] Cam Conigli: Online
[6/12/2020, 23:30:00] [CameraUI] [Interface] Cam Terrazzo: Online
[6/12/2020, 23:30:07] [CameraUI] [Interface] Cam Conigli: RTSP stream exited with error! (null)
[6/12/2020, 23:30:08] [CameraUI] [Interface] Cam Conigli: Online
[6/12/2020, 23:30:10] [CameraUI] [Interface] Cam Terrazzo: RTSP stream exited with error! (null)
[6/12/2020, 23:30:12] [CameraUI] [Interface] Cam Conigli: RTSP stream exited with error! (null)

**Comment by platini76** [comment]
I had them working on web interface with this config:

                {
                    "name": "Cam Conigli",
                    "manufacturer": "Ezviz",
                    "model": "Ezviz Mini Plus",
                    "serialNumber": "xxxxxx",
                    "unbridge": true,
                    "videoConfig": {
                        "source": "-i rtsp://xxxxx@192.168.0.147/Streaming/Channels/1",
                        "stillImageSource": "-i rtsp://xxxxxx@192.168.0.147/Streaming/Channels/1/picture",
                        "maxStreams": 2,
                        "maxWidth": 1920,
                        "maxHeight": 1080,
                        "maxFPS": 15,
                        "maxBitrate": 500,
                        "vcodec": "h264_mmal",
                        "packetSize": 564,
                        "audio": true,
                        "socketPort": 8201
                    }

but video won't work in homekit, I have only still image

and then 

[7/12/2020, 00:15:15] [CameraUI] [Plugin] Cam Conigli: Sending snapshot: 1280 x 720
[7/12/2020, 00:15:16] [CameraUI] [Plugin] Cam Conigli: Starting video stream: 1280 x 720, 15 fps, 299 kbps
[7/12/2020, 00:15:18] [Broadlink RM] Condizionatore Soggiorno onTemperature (0)

**Comment by platini76** [comment]
I have another log:

[7/12/2020, 02:53:18] [CameraUI] [Plugin] Cam Conigli: Stream command: ffmpeg -i rtsp://admin:XXXXXXXX@192.168.0.147/Streaming/Channels/1 -an -sn -dn -codec:v h264_omx -pix_fmt yuv420p -color_range mpeg -r 15 -f rawvideo -filter:v scale='min(1280,iw)':'min(720,ih)':force_original_aspect_ratio=decrease,scale=trunc(iw/2)*2:trunc(ih/2)*2 -b:v 299k -payload_type 99 -ssrc 8720021 -f rtp -srtp_out_suite AES_CM_128_HMAC_SHA1_80 -srtp_out_params t5o5A8k7ot/zCMjXTnjggI8BKCV5cAwf1+wGpgkZ srtp://192.168.0.39:55034?rtcpport=55034&pkt_size=564 -vn -sn -dn -codec:a libfdk_aac -profile:a aac_eld -flags +global_header -f null -ar 16k -b:a 24k -ac 1 -payload_type 110 -ssrc 10164289 -f rtp -srtp_out_suite AES_CM_128_HMAC_SHA1_80 -srtp_out_params suv1ljD61xQ7MsA3mNBwTY2TExxOMSPNdcAzDxMU srtp://192.168.0.39:63493?rtcpport=63493&pkt_size=188
[7/12/2020, 02:53:21] [CameraUI] [Plugin] Cam Conigli: ffmpeg exited with code: 1 and signal: null (Error)


I have also a question, it is correct that when I set the option "unbridge camera" all camera from web interface disappers?

**Comment by SeydX** [comment]
@platini76 

Yes, ``unbridged`` will let them disappear from webinterface. 

It seems, the stream in homekit fails because of enabled audio. You can try with ``"audio": false`` in config.

The issue with audio is already known https://github.com/SeydX/homebridge-camera-ui/issues/65

**Comment by platini76** [comment]
notpe... also sisabling audio wond' start

**Comment by SeydX** [comment]
@platini76 

Your previous config and your actual config are different, can you try with the parameter from previous config?

### Previous config

```
...
"vcodec": "h264",
"source": "-rtsp_transport tcp -re -i rtsp://XXXXXXXX@192.168.0.58/h264_stream",
"debug": true,
"maxFPS": 30,
"maxHeight": 720,
"maxStreams": 2,
"maxWidth": 1280,
"audio": true
...
```

### Actual config

```
...
"source": "-i rtsp://xxxxx@192.168.0.147/Streaming/Channels/1", <— different
"stillImageSource": "-i rtsp://xxxxxx@192.168.0.147/Streaming/Channels/1/picture",
"maxStreams": 2,
"maxWidth": 1920, <— different
"maxHeight": 1080,  <— different
"maxFPS": 15,  <— different
"maxBitrate": 500,  <— not defined in previous config
"vcodec": "h264_mmal", <— different
"packetSize": 564, <— not defined in previous config
"audio": true,
"socketPort": 8201
...
```


Can you try with following config:

```
"videoConfig": {
    "source": "-rtsp_transport tcp -re -i rtsp://XXXXXXXX@192.168.0.58/h264_stream",
    "stillImageSource": "-rtsp_transport tcp -re -i rtsp://XXXXXXXX@192.168.0.58/h264_stream",
    "maxStreams": 4, 
    "maxWidth": 1280,
    "maxHeight": 720,
    "maxFPS": 30,
    "vcodec": "h264",
    "audio": false,
    "socketPort": 8201
}
```

**Comment by platini76** [comment]
now it seems working... without audio...

it was a problem on resolution? I copy your rest on the same camera to do it.
any possibility ti transmit @ 1920x1080 as camera are capable? I know the new eufy can do it directly in homekit.

and... possibility to unbridge camera and see also them on web interface?
cause I have a virtual machine and I don't want to have it hevvy

thanks

**Comment by SeydX** [comment]
@platini76 

Atm it is not possible to see unbridged cameras via webinterface. This will be available in a next version. 

You can play with resolution. Camera UI accepts all parameter which camera-ffmpeg also accepts

**Comment by platini76** [comment]
can you say me the max for homekit=?
it is 1920x1080 or 720?
when nest version?

**Comment by platini76** [comment]
I have several problems with all ezviz camera. all camera video streaming is delayed... and take only some seconds a minute... with big jump... what d you suggest? camera on the ezviz app are perfect.
it is the codec?
it is raspberry processor?
really don't know how to solve.

**Comment by SeydX** [comment]
Unbridged cameras are now available in the ui (v3.2.2)
